service/build.gradle

```diff
@@ -194,9 +194,8 @@ dependencies {
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-resource-server'
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-jose'
   compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.17.1' // override to fix vulnerability issue (CVE-2019-17195)
-  compile group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.7.2'
-  compile group: 'com.launchdarkly', name: 'okhttp-eventsource', version: '1.11.0'
-  compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '4.14.0', classifier: 'thin'
```

Comment:
removed `classifier: thin` so we are not forced to use `gson`. See [here](https://docs.launchdarkly.com/sdk/server-side/java/migration-4-to-5#understanding-changes-to-the-json-value-type)

Reply:
I am not sure if I understood your rationing correctly, I don't see how `thin` is "forcing" us to use `gson`. Please see [here]https://docs.launchdarkly.com/sdk/server-side/java/migration-4-to-5#using-other-third-party-dependencies)
Now Gson is simply embedded in the library and we potentially use different version, but I think I am happy with it, if it works I don't see reason to change especially if we can remove one of our dependences.